tests/integration/test_integration.py

```diff
@@ -731,8 +731,8 @@ async def test_given_application_is_deployed_and_related_to_s3_integrator_when_c
         status = await ops_test.model.get_status()
         minio_ip = (
             status.applications[MINIO_APPLICATION_NAME]
-            .units[f"{MINIO_APPLICATION_NAME}/0"]
-            .address
+            .units[f"{MINIO_APPLICATION_NAME}/0"]  # type: ignore[reportOptionalMemberAccess]
+            .address  # type: ignore[reportOptionalMemberAccess]
         )
```

Comment:
We could do something like:
```
minio_app = status.applications[MINIO_APPLICATION_NAME]
assert minio_app
minio_unit = minio_app.units[f"{MINIO_APPLICATION_NAME}/0"]
assert minio_unit
minio_ip = minio_unit.address
```
And avoid the ignores

Reply:
Thanks, I applied the suggested changes.